components/caseStudyCard.tsx

```diff
@@ -16,25 +16,27 @@ export default function CaseStudyCard({
       className="group block rounded-xl overflow-hidden focus:outline-none"
       href="#"
     >
-      <div className="flex flex-col sm:flex-row sm:items-center gap-3 sm:gap-5">
-        <div className="shrink-0 relative overflow-hidden w-full sm:w-56 ">
-          <div className="shrink-0 relative rounded-xl overflow-hidden w-full sm:w-56 h-44">
+      <span className="flex flex-col sm:flex-row sm:items-center gap-3 sm:gap-5">
```

Comment:
why have you replaced div with span here any specific reason? since we are styling as container here div suits better, for inline styling span is used.
Also since `< a> `itself is styled as a block-level element, it can contain block-level children like `< div>.`

Reply:
When i was resolving hydration issue, the error was saying <div> should not be inside <a> tag, but now it's not coming, I have reverted it.